src/pages/studyView/charts/categoryBarChart/CategoryBarChartAxisLabel.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { VictoryLabel, Helpers } from 'victory';
+import { Tooltip, OverlayTrigger } from 'react-bootstrap';
+
+export class CategoryBarChartAxisLabel extends VictoryLabel {
```

Comment:
same as above. i think we have very similar components with our BarChart component. it would be nice to reuse this somehow. lets look into this for all similarly related components